README.md

```diff
@@ -1,7 +1,6 @@
 ## Web Server Middleware for Dart
 
-[![Build Status](https://travis-ci.org/dart-lang/shelf.svg?branch=master)](https://travis-ci.org/dart-lang/shelf)
-[![Coverage Status](https://coveralls.io/repos/dart-lang/shelf/badge.svg?branch=master)](https://coveralls.io/r/dart-lang/shelf)
```

Comment:
Are we losing coverage checks, or was that not working before?

Reply:
It hasn't been wired up for a while!

https://coveralls.io/github/dart-lang/shelf

Jan 3, 2017!

Reply:
FYI, dartdoc has a coveralls integration working on actions: dart-lang/dartdoc#2464.

Reply:
Happy to get that going in a follow-up!